src/vs/code/electron-main/window.ts

```diff
@@ -424,24 +424,36 @@ export class CodeWindow extends Disposable implements ICodeWindow {
 			this.dispose();
 		});
 
-		// Prevent loading of svgs
-		this._win.webContents.session.webRequest.onBeforeRequest(null!, (details, callback) => {
-			if (details.url.indexOf('.svg') > 0) {
+		// Prevent loading of remote svgs
+		this._win.webContents.session.webRequest.onBeforeRequest((details, callback) => {
+			if (details.url.endsWith('.svg')) {
 				const uri = URI.parse(details.url);
-				if (uri && !uri.scheme.match(/file/i) && uri.path.endsWith('.svg')) {
+				if (uri && uri.scheme.toLowerCase() !== 'file') {
```

Comment:
Use scheme constants from https://github.com/microsoft/vscode/blob/master/src/vs/base/common/network.ts#L41 

`Schemas.file` and also should allow for `Schemas.vscodeFileResource`